.idea/cf-k8s-api.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Curious about the decision to check in this `.idea` config vs `.gitignore`-ing it. Is this so folks using IntelliJ-family editors can have consistent project config?

Reply:
This was an oversight.